dataclasses_avroschema/faker.py

```diff
@@ -1,3 +1,47 @@
-from faker import Faker
+class FakeStub:
```

Comment:
I think it is better to move the class and `_FAKER_METHODS_TO_STUB` inside the `except` block. The reason is to make it more clear and second to load this code only when the exception occurs

Reply:
that's an excellent suggestion. Thank you.